config/clients/java/template/client-ClientWriteResponse.java.mustache

```diff
@@ -10,12 +11,21 @@ public class ClientWriteResponse {
     private final Map<String, List<String>> headers;
     private final String rawResponse;
 
+    // how to better express no call has been made?
```

Comment:
Thanks for the contribution and calling this out, great question! On first glance it does seem a bit odd to return a `200` to the caller when no request was actually made, that could be confusing for customers since no request was actually made? But not sure the best alternative, need to think on it more. @rhamzeh do you have any thoughts on how the SDKs should respond in this case?

Reply:
Same thoughts here (reason for the comment). Current state of mind:

1. To distinguish call/no call, I could think of a more complicated ClientWriteResponse and e.g. fields/types for call/no call? At least the openapi-spec has a single return type. But this wouldn't that lead to a more complicated response handling on consumer side?
2. Argument against this could be: its fine like this, because the caller would not bother. 
3. Could also be an explicit exception, because the call of the writes method makes no sense if no writes or deletes are given? No, I think an exception is to much here.

I try to imagine how the client would handle the result. Would it be of interest if a call has been made or not?